tests/tracer/test_logging.py

```diff
@@ -0,0 +1,284 @@
+import os
+
+
+def test_unrelated_logger_in_debug_with_patch_all(run_python_code_in_subprocess, tmpdir):
+    """
+    When importing the tracer after logging has been configured,
+                with debug mode and enabling a custom log path,
+                the ddtrace logger does not override any custom logs settings and
+                the ddtrace logger continues to have its expected configuration and
+                logs to a custom file path.
```

Comment:
```suggestion
        with debug mode and enabling a custom log path,
            the ddtrace logger does not override any custom logs settings and
            the ddtrace logger continues to have its expected configuration and
            logs to a custom file path.
```

(I am just playing around, feel free to dismiss this comment)

Reply:
I'm going to mark this as resolved since the formatting looks a little different in the latest commit, but if it needs to be changed, I'm open to suggestions!